content/en/profiler/profile_types.md

```diff
@@ -158,20 +158,21 @@ CPU
 Wall Time
 : The elapsed time used by each function. Elapsed time includes time when code is running on CPU, waiting for I/O, and anything else that happens while the function is running.
 
-Allocations (alpha, v1.19.0+)
+Allocations (beta, v1.21.1+)
 : The number of objects allocated by each method during the profiling period (default: 60s), including allocations which were subsequently freed. This is useful for investigating garbage collection load.<br />
-_Requires: Ruby 2.7+_ and [needs to be enabled][2]
+_Requires: [Enabling manually][3]
 
-Heap Live Objects (alpha, v1.19.0+)
+Heap Live Objects (alpha, v1.21.1+)
 : The number of objects allocated by each method in heap memory that have not yet been garbage collected. This is useful for investigating the overall memory usage of your service and identifying potential memory leaks.<br />
 _Requires: Ruby 2.7+_ and [needs to be enabled][2]
```

Comment:
If we change the above from `needs to be enabled`, I'd say we should update these as well?

Reply:
Fixed in 933e1a7066b1afe3c7e30f840178daba183d6f3c